From 35bf03f60abc3b8673710b0d58925e1303674875 Mon Sep 17 00:00:00 2001
From: Gokul Sanker V G <gokul.sanker.v.g@intel.com>
Date: Thu, 1 Dec 2022 16:48:28 +0530
Subject: [PATCH] U-Boot: Disable unused zlib functionality

U-Boot uses its own zlib derived from the old, zlib_1.2.3 version.
Since we are not using any functionlaity of zlib, it can be disabled.
This can avoid the need for addressing BDBA scan issues in this code.
In future, if we are going for any zlib functionality such as handling
compressed image format, we need to enable these macros.

Signed-off-by: Gokul Sanker V G <gokul.sanker.v.g@intel.com>
Co-authored-by: AppaRao Puli <apparao.puli@intel.com>
Upstream-Status: Pending
---
 include/config_defaults.h | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/include/config_defaults.h b/include/config_defaults.h
index 7ef928bbe1..f51aa49b97 100644
--- a/include/config_defaults.h
+++ b/include/config_defaults.h
@@ -16,7 +16,12 @@
 #define CONFIG_BOOTM_RTEMS 1
 #define CONFIG_BOOTM_VXWORKS 1
 
-#define CONFIG_GZIP 1
-#define CONFIG_ZLIB 1
+/*
+ * Disabling zlib functionality until source
+ * is updated in community to recent
+ * or this functionality is must required.
+ */
+//#define CONFIG_GZIP 1
+//#define CONFIG_ZLIB 1
 
 #endif
-- 
2.25.1

